python/perforce.py

```diff
@@ -1,8 +1,10 @@
 """
 Manage a perforce workspace in the context of a build machine
 """
+from functools import wraps
 import os
 import re
+import signal
```

Comment:
It doesn't look like we actually used `signal` or `wraps`.

Reply:
Removed